src/include/binder/copy/bound_copy.h

```diff
@@ -13,12 +13,11 @@ namespace binder {
 
 class BoundCopy : public BoundStatement {
 public:
-    BoundCopy(
-        common::CopyDescription copyDescription, common::table_id_t tableID, std::string tableName)
+    BoundCopy(const common::CopyDescription& copyDescription, common::table_id_t tableID,
+        std::string tableName)
```

Comment:
If we pass copyDescription by const reference, we should also pass tableName by reference to be consistent.